### PR TITLE
I/6137: Brought table properties and table cell properties to the feature guide

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Brought table properties and table cell properties to the feature guide. Closes ckeditor/ckeditor5#6137.

### CODE REVIEW

**Comment by Reinmar** [comment]
cc @AnnaTomanek

File: docs/_snippets/features/table-styling.html [hunk 1]
```diff
@@ -1,54 +1,76 @@
...
+	<figure class="table">
+		<table style="border-bottom:1px solid;border-left:1px solid;border-right:1px solid;border-top:1px solid;">
+			<thead>
+				<tr>
+					<th style="background-color:#eee;width:317px;" colspan="1" rowspan="2"><p>&nbsp;</p><p>&nbsp;</p></th>
+					<th style="background-color:#1d7373;text-align:center;width:224px;" colspan="3">Full Year (unaudited)</th>
+					<th style="background-color:#1d7373;text-align:center;width:360px;" colspan="4">2013 (unaudited)</th>
+				</tr>
+				<tr>
+					<th style="background-color:#5ccccc;width:78px;">2011</th>
```
Comment (Reinmar) [inline docs/_snippets/features/table-styling.html]: Shouldn't we use colors that are available in out colorpicker? So e.g. someone may reuse the same color?

File: docs/_snippets/features/table-styling.html [hunk 2]
```diff
@@ -1,54 +1,76 @@
...
+					<td style="width:72px;">$6,526</td>
+					<td style="width:72px;">$6,792</td>
+					<td style="width:72px;">$7,083</td>
+					<td style="width:72px;">$8,098</td>
+				</tr>
+				<tr>
+					<th style="background-color:#ffd5b2;text-align:center;width:317px;"><strong>Property and Equipment, Net</strong></th>
+					<td style="width:78px;">$9,603</td>
+					<td style="width:73px;">$11,854</td>
+					<td style="width:72px;">$16,524</td>
```
Comment (jodator) [inline docs/_snippets/features/table-styling.html]: OCD is kicking in :D

Comment (Reinmar) [reply docs/_snippets/features/table-styling.html]: 👍

**Comment by oleq** [comment]
I'm adding a new section about configs so please postpone the merge.

**Comment by oleq** [comment]
There you go @AnnaTomanek. It should be way better now.

**Comment by AnnaTomanek** [comment]
Thanks a lot :clap:. Fixed some really minor stuff and we're good to go!

**Pull request merged**